modules/check-ownership.js

```diff
@@ -1,7 +1,7 @@
 const cp = require('child_process')
       path = require('path');
 
-module.exports = (details, publish) => {
+module.exports.checkOwnership = (details, publish) => {
```

Comment:
What is the reason for using a named export here instead of the default like it was using before? Doesn't seem to add any value?

Reply:
I don't know any other ways to make it mockable.

Reply:
did some digging and thinking and decided this will work